flocx_market/templates/index.html

```diff
@@ -0,0 +1,34 @@
+<%inherit file="layout.html" />
```

Comment:
We probably don't need this: this service implement a REST api, not a web UI.

Reply:
removed and made a new PR